### DISCUSSION TITLE
Ellipsis 组件这里为什么要加 0.5？有多少行不是乘多少行吗，最好加下注释

### DISCUSSION THREAD

**Discussion by linxianxi** [ant-design/ant-design-mobile#5722] [General]
链接： https://github.com/ant-design/ant-design-mobile/commit/4a0eae9830eb3b64e29056e6ea6266460d28ce48

**Comment by miracles1919** [comment]
https://github.com/ant-design/ant-design-mobile/issues/4428

**Reply by linxianxi** [reply]
没懂为什么要加一个乘0.5

**Reply by miracles1919** [reply]
因为会有计算上的误差，会导致内容没了，看上面发的 issue
